games-arcade/moleinvasion/moleinvasion-0.4.recipe

```diff
@@ -12,7 +12,8 @@ PATCHES="moleinvasion-0.4.patch"
 ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
 
 PROVIDES="
-	cmd:moleinvasion = $portVersion
+	moleinvasion
```

Comment:
My bad, sorry.
Il 01/lug/2016 20:10, "Jérôme Duval" <notifications@github.com> ha scritto:

> In games-arcade/moleinvasion/moleinvasion-0.4.recipe
> <https://github.com/haikuports/haikuports/pull/672#discussion_r69336765>:
>
> > @@ -12,7 +12,8 @@ PATCHES="moleinvasion-0.4.patch"
> >  ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
> >
> >  PROVIDES="
> > -	cmd:moleinvasion = $portVersion
> > +	moleinvasion
>
> "= $portVersion" is missing.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/haikuports/haikuports/pull/672/files/a5cae98415e332a0f2b62dba2e1f3819b4775ff6#r69336765>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA4qbXuq70LuiGB4SyUkTgIVqynbEIO9ks5qRVgjgaJpZM4JDVpG>
> .
>